skyportal/facility_apis/fake_request.py

```diff
@@ -0,0 +1,33 @@
+from tornado import httputil
+
+
+class FakeApplication(object):
+    def __init__(
+        self,
+    ) -> None:
```

Comment:
not type annotating!

Reply:
I just removed this, which was based loosely on what tornado.web had.